sql/_13_issues/_12_1h/cases/bug_bts_6768.sql

```diff
@@ -7,8 +7,10 @@ CREATE INDEX i_bugs_subject  ON bugs(Subject) WHERE Subject is not null;
 CREATE INDEX idx1 ON bugs(ln(bugID));
 CREATE INDEX idx2 ON bugs(ln(Closed));
 CREATE INDEX idx3 ON bugs(ln(bugID),Closed);
+update statistics on all classes;
```

Comment:
change 'all classes' to specific classes. The 'all classes' can affect other test cases as well.

Reply:
All changed to specific classes.